gtsfm/visualization/view_scene.py

```diff
@@ -63,7 +64,35 @@ def view_scene(args: argparse.Namespace) -> None:
     for i in range(len(wTi_list)):
         wTi_list[i] = zcwTw.compose(wTi_list[i])
 
-    draw_scene_open3d(point_cloud, rgb, wTi_list, calibrations, args)
+    if args.gt_dir is not None:
+        # Plot both scene data, and GT data, in same coordinate frame.
+        # Read in GT data.
+        wTi_list_gt, _, gt_calibrations, _, _, _ = io_utils.read_scene_data_from_colmap_format(args.gt_dir)
+
+        if len(gt_calibrations) == 1:
+            gt_calibrations = gt_calibrations * len(img_fnames)
+
+        for i in range(len(wTi_list_gt)):
+            wTi_list_gt[i] = zcwTw.compose(wTi_list_gt[i])
+
+        # Align the poses.
+        n = min(len(wTi_list), len(wTi_list_gt))
+        wTi_aligned_list, rSe = alignment_utils.align_poses_sim3_ignore_missing(wTi_list_gt[:n], wTi_list[:n])
```

Comment:
How would this work if the missing cameras are towards the start of the sequence.

Reply:
@ayushbaid good catch, fixed now